pulp_smash/utils.py

```diff
@@ -159,8 +159,8 @@ def execute_pulpcore_python(cli_client, command):
         bin_dir = "/usr/local/bin"
         for line in ps_output:
             if "pulpcore" in line:
-                if "bin/python" in line.split()[4]:
-                    bin_dir = line.split()[4].rsplit("/", maxsplit=1)[0]
+                if "bin/python" in line.split()[5]:
+                    bin_dir = line.split()[5].rsplit("/", maxsplit=1)[0]
```

Comment:
Assuming this has worked for someone before, the format of ps must have chaged. Can we use something more stable? Maybe `ps ax -o command`?

Reply:
This command was never run inside the container before. It was only run inside our developer VMs. This change continues to work inside a VM and also inside the centos based container.